### PR TITLE
SYNCT-132: "AtomFeed module doesn't cooperate with Event module in eS…

### PR DESCRIPTION
**Pull request by tomaszmueller**
…aude"

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/atomfeed/api/db/hibernate/HibernatingEventInterceptor.java [hunk 1]
```diff
@@ -39,7 +39,7 @@
  */
 
 @Component
-public class HibernateEventInterceptor extends EmptyInterceptor {
+public class HibernatingEventInterceptor extends EmptyInterceptor {
```
Comment (pgesek) [inline api/src/main/java/org/openmrs/module/atomfeed/api/db/hibernate/HibernatingEventInterceptor.java]: The class name should rather remain the same, change the bean name using something like:

@Component("atomfeed-hibernate-interceptor")

Comment (pgesek) [reply api/src/main/java/org/openmrs/module/atomfeed/api/db/hibernate/HibernatingEventInterceptor.java]: Eventually you can change the classname to AtomFeedHibernateEventInterceptor

**Comment by tomaszmueller** [comment]
@pgesek corrected

**Comment by tomaszmueller** [comment]
can somebody please merge this if possible
@dkayiwa, @raff ?

**Pull request merged**